test/integration-test.py

```diff
@@ -241,6 +248,45 @@ def run_client_tests():
     cmd = os.path.join(root, 'tests', 'boulder-integration.sh')
     run_custom(cmd, cwd=root)
 
+def run_expired_authz_purger_test(challenge_types):
+    if run_node_test("challenge-not-started.com", challenge_types[0], None, "startChallenge")\
+       != ABORT_SUCCESS:
+        print("\nDidn't cleanly abort before attempting challenge")
+        die(ExitStatus.NodeFailure)
+
+        now = datetime.datetime.now()
+        after_grace_period = now + datetime.timedelta(days=+14, minutes=+3)
+
+        target_time = now
+        expected_output = 'Deleted a total of 0 expired pending authorizations'
+        try:
+            out = subprocess.check_output(
+                '''FAKECLOCK=`date -d "%s"` ./bin/expired-authz-purger --config cmd/expired-authz-purger/config.json --yes'''
```

Comment:
I still feel like we should abstract out this "run with FAKECLOCK" concept into its own function. We don't need to modify the existing use of FAKECLOCK in this PR, but at least these two new places should use a better approach.